### PR TITLE
Run core security tests against api

### PR DESCRIPTION
**Pull request by jrafanie**
The hope is we can detect possible security issues before PRs are merged.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -47,6 +47,8 @@ jobs:
       run: bin/setup
     - name: Run tests
       run: bundle exec rake
+    - name: Run core security tests
+      run: bundle exec rake spec:security
```
Comment (jrafanie) [inline .github/workflows/ci.yaml:51]: Note, this repo doesn't have suites so I added this step to run after running the normal tests.

**Comment by kbrock** [comment]
love this. Can't wait until we square away brakeman for our suite

I guess we do depend upon automate
But content?
and ui-classic?

**Comment by jrafanie** [comment]
We want this backported, just not right now

**Comment by jrafanie** [comment]
This successfully detected the possible command injection added in https://github.com/ManageIQ/manageiq-ui-classic/pull/8686, and since https://github.com/ManageIQ/manageiq-ui-classic/pull/8705 has been merged, removing that issue, I've rekicked this PR.

**Comment by jrafanie** [comment]
> This successfully detected the possible command injection added in [ManageIQ/manageiq-ui-classic#8686](https://github.com/ManageIQ/manageiq-ui-classic/pull/8686), and since [ManageIQ/manageiq-ui-classic#8705](https://github.com/ManageIQ/manageiq-ui-classic/pull/8705) has been merged, removing that issue, I've rekicked this PR.

@Fryguy this is 💚 too

**Pull request merged**